ledger-core/reference/local.go

```diff
@@ -215,6 +215,11 @@ func (v Local) Size() int {
 	return BinarySizeLocal(v)
 }
 
+// deprecated: use reference.MarshalLocalToSizedBuffer
+func (v Local) MarshalToSizedBuffer(data []byte) (int, error) {
```

Comment:
:O new method that is already deprecated?

Reply:
It's needed for GoGo Protobuf

Reply:
yes, we have to move to use containers in messages to get rid of it, but this is R1